app/src/main/java/com/codesoom/assignment/handler/TaskHandler.java

```diff
@@ -0,0 +1,27 @@
+package com.codesoom.assignment.handler;
+
+import com.sun.net.httpserver.HttpExchange;
+import com.sun.net.httpserver.HttpHandler;
+
+import java.io.IOException;
+import java.io.OutputStream;
+import java.nio.charset.StandardCharsets;
+
+public class TaskHandler implements HttpHandler {
+
+    public final int SUCCESSFUL_OK = 200;
+    @Override
+    public void handle(HttpExchange exchange) throws IOException {
+        String method = exchange.getRequestMethod();
+        String path = exchange.getRequestURI().getPath();
```

Comment:
이런 값들은 변경되지 않도록 해주는 것도 실수를 예방하는 방법이 됩니다.

```suggestion
        final String method = exchange.getRequestMethod();
        final String path = exchange.getRequestURI().getPath();
```

Reply:
넵 감사합니다 ㅎㅎ